src/main/kotlin/app/revanced/utils/adb/Adb.kt

```diff
@@ -15,6 +15,9 @@ internal class Adb(
     private val logging: Boolean = true
 ) {
     private val device: JadbDevice
+    companion object {
+        var SuperSU: Boolean = true
```

Comment:
```suggestion
        var usingSuperSU: Boolean = true
```

Reply:
Usually, you'd even use an enum for this.

Reply:
Also this suggestion has to be batched with others.

Reply:
> Usually, you'd even use an enum for this.

What exactly do you mean? How'd using a enum be possible here?

Reply:
The reason is that you are switching between root modes such as superSU and Magisk and maybe X in the future, this is calling for an enum.

Reply:
> The reason is that you are switching between root modes such as superSU and Magisk and maybe X in the future, this is calling for an enum.

I'll update it to instead be a Enum and mark this pr ready when I've done that and tested it.

Reply:
Sounds good, thanks for bearing with our changes 👍